client/asset/btc/btc.go

```diff
@@ -1029,6 +1050,7 @@ func (btc *baseWallet) Connect(ctx context.Context) (*sync.WaitGroup, error) {
 	if err := btc.node.connect(ctx, &wg); err != nil {
 		return nil, err
 	}
+	btc.ctx = ctx
```

Comment:
Have you finished this FeeEstimator rework? Lmk when it is ok to come back to this issue

Reply:
You can go ahead.  I want another direction instead of changing it.  I don't like the RawRequestor part of it, but I just worked around it entirely in the Electrum PR that I was alluding to.